js/notify/settings/configure-alert-position.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```

Comment:
I know we won't have the title showing up in our child window, but can you please change this to "Configure Notification Position"?